learn/classes/static_class.md

```diff
@@ -242,6 +314,8 @@ Static variables are shared among all instances of a class and can be accessed u
 
 ## The Difference between a Class Method and a Static Method
 
+Now that you're familiar with class methods and static methods, it's time to delve into the world of static class python and its distinctive properties.
```

Comment:
1. Python should start from upper-case.
2. "static class python" doesn't sound good. Let's rephrase it.